docs/standard/glossary.md

```diff
@@ -0,0 +1,267 @@
+﻿---
+title: .NET Glossary
+description: Find out the meaning of selected terms used in the .NET documentation.
+keywords: .NET glossary, .NET dictionary, .NET terminology, .NET platform, .NET framework, .NET runtime
+author: tdykstra
+ms.author: tdykstra
+ms.date: 07/08/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: dotnet-standard
+ms.devlang: dotnet
+---
+
+# .NET Glossary
+
+The primary goal of this glossary is to clarify meanings of selected terms and acronyms that appear frequently in the .NET documentation without definitions.
+
+## AOT
+
+Ahead-of-time compiler.
+
+Similar to [JIT](#jit), this compiler also translates [IL](#il) to machine code. In contrast to JIT compilation, AOT compilation happens before the application is executed and is usually performed on a different machine. Because AOT tool chains don't compile at runtime, they don't have to minimize time spent compiling. That means they can spend more time optimizing. Since the context of AOT is the entire application, the AOT compiler also performs cross-module linking and whole-program analysis, which means that all references are followed and a single executable is produced.
+
+## ASP.NET 
+
+The original ASP.NET implementation that ships with the .NET Framework.
+
+Sometimes ASP.NET is an umbrella term that refers to both ASP.NET implementations including ASP.NET Core. The meaning that the term carries in any given instance is determined by context. 
+
+See [ASP.NET](/aspnet/#pivot=aspnet).
+
+## ASP.NET Core
+
+A cross-platform, high-performance, open source implementation of ASP.NET built on .NET Core.
+
+See [ASP.NET Core](/aspnet/#pivot=core).
+
+## CLR
+
+Common language runtime.
+
+The exact meaning depends on the context, but this usually refers to the runtime of the .NET Framework. The CLR handles memory allocation and management. The CLR is also a virtual machine that not only executes apps but also generates and compiles code on-the-fly using a JIT compiler. The current Microsoft CLR implementation is Windows only.
+
+## CoreCLR
+
+.NET Core common language runtime.
+
+This CLR is built from the same code base as the CLR. Originally, CoreCLR was the runtime of Silverlight and was designed to run on multiple platforms, specifically Windows and OS X. CoreCLR is now part of .NET Core and represents a simplified version of the CLR. It's still a cross platform runtime. CoreCLR is also a virtual machine with JIT and code execution capabilities.
```

Comment:
Somewhere in this paragraph, we need to mention Linux. Suggestion:

"It's still a cross platform runtime." => "It's still a cross platform runtime, now including support for many Linux distributions."